### PR TITLE
Fix/docs

### PR DESCRIPTION
**Pull request by iparask**
It compiles correctly to docs for version 1.0.0

### CODE REVIEW

**Comment by iparask** [comment]
This will trigger will appear when it is merged with master.

**Comment by mturilli** [comment]
Before I start the review, I am wondering why we should change the theme/formatting of the documentation. Are we solving any functional issue? If not, seeing the sorry state of the *content* of the documentation and the fact that if we change the theme of RP documentation we need to change it for all the RCT, I would focus on the content and leave the formatting as it is.

**Comment by iparask** [comment]
The changes in the css files are because I updated the sphinx-config files. This version of the docs can be seen [here](https://radicalpilot.readthedocs.io/en/fix-docs/)

File: docs/source/index.rst [hunk 1]
```diff
@@ -88,6 +87,5 @@ Contents:
    benchmarks.rst
    faq.rst
    developer.rst
+   release_notes.rst
```
[the remark this answers is not in the document]
Comment (iparask) [reply docs/source/index.rst]: I am not sure how to do it. I think I can have it in a completely separate page, where the list is not visible at all and when people click on the release note they will be able to see the whole thing.

[the remark this answers is not in the document]
Comment (mturilli) [reply docs/source/index.rst]: Sounds good enough to me :)

**Comment by andre-merzky** [comment]
Thanks a lot, Giannis!

**Pull request merged**